package.json

```diff
@@ -20,28 +20,27 @@
     "testonly": "jest"
   },
   "dependencies": {
-    "http-status": "^1.2.0"
+    "http-status": "^1.4.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.5",
-    "babel-jest": "^23.4.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-config-yoctol-base": "^0.16.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-prettier": "^2.6.2",
-    "http-errors": "^1.6.3",
-    "husky": "^0.14.3",
-    "jest": "^23.4.0",
+    "@babel/plugin-proposal-class-properties": "^7.7.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
+    "babel-eslint": "^10.0.3",
+    "eslint": "^6.7.1",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-config-yoctol-base": "^0.21.2",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-sort-imports-es6-autofix": "^0.5.0",
+    "http-errors": "^1.7.3",
+    "husky": "^3.1.0",
```

Comment:
husky settings need to be migrated
https://github.com/typicode/husky#upgrading-from-014

Reply:
fixed